ruby.js

```diff
@@ -42,8 +42,25 @@ const langConfig = {
 		["[", "]"],
 		["(", ")"]
 	],
+	autoClosingPairs: [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["\"", "\""],
+		["'", "'"],
+		["`", "`"]
+	],
+	surroundingPairs: [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["\"", "\""],
+		["'", "'"],
+		["`", "`"]
+	],
 	indentationRules: {
-		increaseIndentPattern: /^\s*((begin|class|def|else|elsif|ensure|for|if|module|rescue|unless|until|when|while)|(.*\sdo\b))\b[^\{;]*$/
+		increaseIndentPattern: /^\s*((after|before|begin|class|context|def|describe|else|elsif|ensure|for|if|it|module|rescue|unless|until|when|while)|(.*\sdo\b))\b[^\{;]*$/,
```

Comment:
`after`, `before` and `describe` are rspec methods, not reserved words. The indentation requirements for them should be caught with the `do` or `{` indenters.